app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class API::UsersController < ApplicationController
+  respond_to :json
+
+  def show
+    @users = User.all
```

Comment:
I think the convention is that `show` will be referencing a single `User`. We may want to name this `index` to be more consistent.